etc/kayobe/kolla.yml

```diff
@@ -116,10 +116,19 @@ kolla_build_blocks:
   sahara_engine_footer: |
     # Bring in hack to the vanilla plugin
     RUN pip install -U --no-deps git+https://github.com/stackhpc/sahara.git@stackhpc/7.0.2.1
+  monasca_api_footer: |
+    # Rebase on top of rocky
+    RUN pip install -U --no-deps git+https://github.com/stackhpc/monasca-api.git@timerange
```

Comment:
Please could we create a PR for this branch into the stackhpc/rocky branch? We can then tag it, like this for example:

https://github.com/SKA-ScienceDataProcessor/alaska-kayobe-config/blob/alaska-prod/etc/kayobe/kolla.yml#L127